src/appLogic.js

```diff
@@ -0,0 +1,29 @@
+// import report from './weatherData';
+
+// const controller = () => {
+//   document.getElementById('form-submit').addEventListener('click', (event) => {
+//     event.preventDefault();
+//     const cityName = document.getElementById('form-text').value;
+//     report(cityName);
+//   });
+// };
```

Comment:
- [ ] Kindly remove this commented code block.

Reply:
Thank you for highlighting, I have removed all the commented code.

Reply:
Installed Web-dev-server